py-polars/tests/unit/dataframe/test_shape.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+
+import polars as pl
+
+
+@pytest.mark.may_fail_auto_streaming
```

Comment:
@orlp FYI

Reply:
@ritchie46 I think we need 'is scalar and thus allowed to broadcast' information into the zip node for that.

Reply:
We have `is_scalar`. Let's discuss tomorrow.